swarm/network/simulation/http_test.go

```diff
@@ -83,27 +84,27 @@ func TestSimulationWithHTTPServer(t *testing.T) {
 	if result.Error != nil {
 		t.Fatal(result.Error)
 	}
+	if err := <-errC; err != nil {
+		t.Fatal(err)
+	}
 	log.Debug("Test terminated successfully")
 }
 
-func sendRunSignal(t *testing.T) {
+func sendRunSignal(t *testing.T, errC chan error) {
```

Comment:
The name 'run signal' is a little bit confusing for me. I noticed I started to think about OS signals, but there is nothing like `RUN` there.

Reply:
Signal is used here in a different sense then os signals and pretty localized to http request in this function. The name of the function is not changed in this PR, but if you have a suggestion for a better name, we can include it in?

Reply:
What about `triggerSimulationRun` as trigger has a more wider meaning, I think?

Reply:
Renamed, thanks for the suggestion.